lib/documentLoader.js

```diff
@@ -2,14 +2,38 @@
  * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
  */
 'use strict';
+/*eslint max-len: ["error", { "ignoreComments": true }]*/
 
 // load locally embedded contexts
 const contexts = require('./contexts');
 
+/**
+ * documentLoader is a set of methods used to dereference jsonld documents.
```

Comment:
Hmm, this is confusing. Here `documentLoader` refers to this little utility module, which is not a set of methods used to dereference jsonld documents. Rather, it is "A utility module that provides a set of functions for using or extending jsonld-signature's built-in JSON-LD document loader." We can add "To learn more about document loaders, see (put link to jsonld.js here) or [The JSON-LD API spec](https://www.w3.org/TR/json-ld11-api/#loaddocumentcallback)".

Reply:
this kind of highlights a point I was making the use of documentLoader in this project creates confusion on which documentLoader is being used. Anyways, that's a better description so working on it.

Reply:
Well, users don't ever use this file. The utilities in this file are exported on the main API. This file is an internal implementation detail for organizational purposes. Perhaps even having a documentation `module` tag here is a bad idea.